web/src/routes/(user)/people/+page.svelte

```diff
@@ -209,6 +293,22 @@
   };
 </script>
 
+{#if showMergeModal}
+  <FullScreenModal on:clickOutside={() => (showMergeModal = false)}>
+    <SuggestMerge
+      bind:personMerge1
+      bind:personMerge2
```

Comment:
Personally, I don't think these should be bound. They should be inputs, and then the suggest merge component can do whatever it wants and emit the final results in the confirm event.